server/app/durablejobs/jobs/AddOperatorToLeafAddressServiceAreaJob.java

```diff
@@ -0,0 +1,141 @@
+package durablejobs.jobs;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import durablejobs.DurableJob;
+import io.ebean.DB;
+import io.ebean.Database;
+import io.ebean.SqlRow;
+import io.ebean.Transaction;
+import io.ebean.TxScope;
+import java.util.List;
+import java.util.Objects;
+import models.PersistedDurableJobModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import services.program.predicate.Operator;
+
+public final class AddOperatorToLeafAddressServiceAreaJob extends DurableJob {
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(AddOperatorToLeafAddressServiceAreaJob.class);
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+
+  private final Database database;
+  private final PersistedDurableJobModel persistedDurableJobModel;
+
+  public AddOperatorToLeafAddressServiceAreaJob(PersistedDurableJobModel persistedDurableJobModel) {
+    this.persistedDurableJobModel = checkNotNull(persistedDurableJobModel);
+    this.database = DB.getDefault();
+  }
+
+  @Override
+  public PersistedDurableJobModel getPersistedDurableJob() {
+    return persistedDurableJobModel;
+  }
+
+  @Override
+  public void run() {
+    LOGGER.debug("Run - Begin");
+
+    String selectSql =
+        """
+SELECT id, block_definitions
```

Comment:
**suggestion:** May be helpful to add a comment here too about what the difference is between these two jsonb paths?

Reply:
Is there something specific you're looking for? I feel like I'm just going to duplicate the query in English.

Reply:
I think the javadoc above has a good summary, so I don't think this needs to be updated. Mainly was just thinking - checks visibility and eligibility predicates for leafAddressServiceArea nodes or something like that